projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -218,4 +220,42 @@ public static function load_marketplace_products_updater() {
 	public static function load_first_posts_stream_helpers() {
 		require_once __DIR__ . '/features/first-posts-stream/first-posts-stream-helpers.php';
 	}
+
+	/**
+	 * Determine whether to disable the comment experience.
+	 *
+	 * @param int $blog_id The blog ID.
+	 * @return boolean
+	 */
+	private function should_disable_comment_experience( $blog_id ) {
+		require_once WP_CONTENT_DIR . '/lib/wpforteams/functions.php';
```

Comment:
Thought: this is going to fail on Atomic sites, as this code only exists on WordPress.com Simple. Or did we already verify that it will work?

Reply:
You're right, it did fail on Atomic sites, leading to the revert in #35194. The second attempt at this in #35196 (specifically e89db3566ffcea92c99d357749e73baa1f4b9aa8) both added a check before the require and added a check to only register the action that calls this method on Simple.